specs/pvs/corla.pvs

```diff
@@ -758,20 +802,23 @@ BEGIN
 
 END roles
 
-canvas_boards: THEORY
+canvass_boards: THEORY
 BEGIN
   IMPORTING roles, election
 
-  canvas_board?: [election -> canvas_board]
-END canvas_boards
+  canvass_board?: [election -> canvass_board]
+END canvass_boards
 
 %+END roles
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-%%% \section{Cryptography}
+%%% \subsection{Cryptography}
 
-%%% We will add additional cryptographic primitives here as required.
+% We will add additional cryptographic primitives here as required.
+% @review kiniry dmz Is it correct to say that the cryptography is used
+% (via hashing) to verify that the ballot manifests and cvrs stored in the 
+% database are actually the ones uploaded by the Counties?
```

Comment:
The hash confirms first of all that the file upload proceeds without any corruption, when it is checked after receipt.  It can also be used by counties, CDOS and the public to confirm that everyone is talking about exactly the same file, after it is downloaded again. It looks like they will be stored as rather large binary objects in Postgres. I hope it and our deployment / design can handle objects that are pushing a gigabyte in size, though we won't see that in this state, this year.